aexpect/session_ops.py

```diff
@@ -0,0 +1,598 @@
+# This Python file uses the following encoding: utf-8
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+
+"""
+
+SYNOPSIS
+------------------------------------------------------
+A list of frequent operations performed through remote sessions.
+
+Copyright: 2018 Intra2net AG
```

Comment:
I guess this is should go after line 12. And you could add your name as Author too.

Reply:
+1

Reply:
I would typically add author but since this module has many different authors whose names are hard to trace and list (I have their permission of course), I will just leave it to the "aexpect contributors". I hope that's ok with you.